qt/icons/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+set(SRC_FILES src/CharIconPainter.cpp src/CharIconEngine.cpp src/Icon.cpp)
+
+set(INC_FILES
+    inc/MantidQtIcons/CharIconPainter.h
+    inc/MantidQtIcons/CharIconEngine.h
+    inc/MantidQtIcons/Icon.h)
+
+qt4_add_resources(RES_FILES ${CMAKE_CURRENT_LIST_DIR}/resources/icons.qrc)
+
+if(ENABLE_MANTIDPLOT)
+  find_package(Qt4 COMPONENTS QtScript REQUIRED)
+  mtd_add_qt_library (TARGET_NAME MantidQtIcons
+                      QT_VERSION 4
+                      SRC ${SRC_FILES}
+                      NOMOC ${INC_FILES}
+                      DEFS
+                        IN_MANTIDQT_ICONS
+                      INCLUDE_DIRS
+                        ${CMAKE_CURRENT_SOURCE_DIR}/inc
+                      LINK_LIBS
+                        ${TCMALLOC_LIBRARIES_LINKTIME}
+                        ${POCO_LIBRARIES}
+                        ${Boost_LIBRARIES}
+                        ${JSONCPP_LIBRARIES}
+                        Qt4::QtScript
+                      INSTALL_DIR_BASE
```

Comment:
I think this needs to go into the `lib` directory alongside `common` as it is linked to by the sip wrapper and not to be loaded by the `InterfaceManager`.  From (and including) the `INSTALL_DIR_BASE` should be 
```
                   INSTALL_DIR
                     ${LIB_DIR}
                   OSX_INSTALL_RPATH
                     @loader_path/../MacOS
                     @loader_path/../Libraries
                   LINUX_INSTALL_RPATH
                     "\$ORIGIN/../${LIB_DIR}"
```